### ISSUE TITLE
One-directional sync

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#940]
It would be useful to make window sync one-directional sometimes.

Related #648

**Comment by agrogers** [comment]
Yeah, i have wanted that also. But we need to be careful that the interface does not get too complicated in order to manage the syncing.
